.github/CODEOWNERS

```diff
@@ -433,3 +433,5 @@ Makefile                                              @thiagokokada
 /modules/programs/ion.nix                             @jo1gi
 
 /modules/services/plex-mpv-shim.nix                   @starcraft66
+
+/modules/services/mpd-discord-rpc.nix                 @Kranzes 
```

Comment:
Please preserve the order in the CODEOWNERS file.

Reply:
What is the order?

Reply:
Got it.